cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx

```diff
@@ -1022,6 +1022,45 @@ export class ToolsControlComponent extends React.PureComponent<Props, State> {
             });
         });
 
+        function checkAttributesCompatibility(
+            functionAttribute: ModelAttribute | undefined, // model
+            dbAttribute: any, // job
+            value: string,
+        ): boolean {
+            if (functionAttribute === undefined || dbAttribute === undefined) {
+                return false;
+            }
+
+            if (functionAttribute.input_type === 'checkbox') {
+                return ['true', 'false'].includes(value);
+            }
```

Comment:
This is an improvement because otherwise this block of code should land in two branches of execution:
1. Inside the if condition on line 1038 where input types are equal.
2. Inside the switch which means the condition for input types equality is not satisfied.
Shifting it up before the condition covers both cases.
Should I add a comment or duplicate this block as described above?

Reply:
I see now. I would suggest to duplicate this code just to improve readability, as it's done to `'number'`